controllers/space/MembershipController.php

```diff
@@ -68,4 +69,51 @@ public function actionIndex($spaceId){
         }
         return $this->returnPagination($query,$pagination,$results);
     }
+
+    /**
+     * @return void
+     * Get all the spaces list of the user
+     */
+    //Get the list of all spaces of the user (issue-8 new endpoint)
+    public function actionGetSpaces(){
+       $userId = Yii::$app->user->id;
+
+       $user = User::findOne($userId);
+
+       if($user == null){
+           return $this->returnError(404, 'User not found!');
+       }
+
+       $spaceMemberships = Membership::find()->where(['user_id' => $userId]);
+
+        if($spaceMemberships->all() == null){
+            return $this->returnError(404, 'Space not found for the user : '.$userId);
+        }
+
+        $results = [];
+        $pagination = $this->handlePagination($spaceMemberships);
+
+        foreach ($spaceMemberships->all() as $spaceMembership){
+            $results[] = $this->getSpacesListData($spaceMembership);
+        }
+
+        return $this->returnPagination($spaceMemberships,$pagination,$results);
+    }
+
+    public function getSpacesListData(Membership $membership){
+        return [
+            'user' => UserDefinitions::getUserShort($membership->user),
```

Comment:
can you please return the owner instead of the current user here?
this is missing, which is available in the `/space`:

```
"owner": {
  "id": 1,
  "guid": "f4122bdab1-3a22ba-4734-b5c0-d3dba5c75",
  "display_name": "A BX,
  "url": "http://...."
}
```

Reply:
Okay @donni106